packages/twig-new/rollup.config.js

```diff
@@ -0,0 +1,79 @@
+import { globSync } from "glob";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import swc from "@rollup/plugin-swc";
+import copy from "rollup-plugin-copy";
+
+import theme from "@ilo-org/themes/tokens/theme/base.json" assert { type: "json" };
```

Comment:
Hey @bashlk, just a note here, if we're eventually going to deprecate this package, where else should this value be set?

Reply:
A quick search of the repo shows that this value is also used to substitute the prefix in the react package and also in the storybook of this package.